packages/devtools_app/lib/src/timeline/flutter_frames_chart.dart

```diff
@@ -57,8 +51,8 @@ class _FlutterFramesChartState extends State<FlutterFramesChart>
   double get availableChartHeight => defaultChartHeight - defaultSpacing;
 
   double get msPerPx =>
-      widget.longestFrameDurationMs.clamp(minMsForDisplay, maxMsForDisplay) /
-      availableChartHeight;
+      // Multiply by two to reach two times the target frame time.
```

Comment:
nit: make comment more clearly describe the why not the how. For example something like:
```/// Milliseconds per pixel value to display a time range spanning two times the target frame rate.```

Reply:
done